libsolidity/codegen/LValue.cpp

```diff
@@ -242,9 +245,17 @@ void StorageItem::storeValue(Type const& _sourceType, SourceLocation const& _loc
 					<< Instruction::DUP2
 					<< Instruction::MUL
 					<< Instruction::DIV;
-			else if (m_dataType->category() == Type::Category::FixedPoint)
+			else if (
+				m_dataType->category() == Type::Category::FixedPoint &&
+				dynamic_cast<FixedPointType const&>(*m_dataType).isSigned()
+			)
 				// implementation should be very similar to the integer case.
-				solAssert(false, "Not yet implemented - FixedPointType.");
+				m_context
+					<< (u256(1) << (8 * (32 - m_dataType->storageBytes())))
```

Comment:
Please do not copy the code from the integer case but rather combine the two cases.